src/structurizr/model/software_system.py

```diff
@@ -62,7 +66,7 @@ def __init__(self, *, location: Location = Location.Unspecified, **kwargs) -> No
         """"""
         super().__init__(**kwargs)
         self.location = location
-        self.containers = set()
+        self.containers: Iterable[Container] = set()
```

Comment:
If you want to annotate the type here it should be set.

```suggestion
        self.containers: Set[Container] = set()
```